utilities/unittests/test_pytest_utils.py

```
@@ -2181,3 +2182,141 @@ def test_validate_cpu_arch_params_called_first(
             mock_generate_common.assert_not_called()
             mock_generate_instance.assert_not_called()
+
+
+class TestAssertIncrementalClassesFullyCollected:
+    """Test cases for assert_incremental_classes_fully_collected function."""
+
+    def test_all_tests_collected_no_error(self):
+        """No error when all tests in an incremental class are collected."""
+
+        class MyClass:
+            def test_one(self): ...
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `test_one`

Add return type annotation: `None`

(ANN202)